client/webserver/site/src/js/app.js

```diff
@@ -222,6 +223,30 @@ export default class Application {
     })
   }
 
+  updateExchangeRegistration (dexUrl, isPaid, confs) {
+    const dex = this.exchanges[dexUrl]
+    dex.feePending = !isPaid
+    dex.confs = confs
```

Comment:
I'd like to see a change to the `Exchange` struct so that this info isn't lost on refresh. In particular, the `FeePending` field should go away, and the confirmation count added as a pointer field. 

```
type Exchange struct {
  ...
  ConfsRequired uint32  `json:"confsReq"`
  RegConfirms   *uint32 `json:"confs"`  
}
```

Where a `nil` `RegConfirms` means the DEX is ready to go.

My recommended solution is to catch the `FeePaymentNote` in `(*Core).notify` (this'll be the first time we use that pattern, but I've been considering it as possibility for other things too), and update the `RegConfirms` field there, but if you see a better way, that's fine too.

You'll will still want to keep these updates to the browser state as is, but you will have the right number of confirmations after a refresh if you do it that way.

Reply:
Note updated recommendation.

Reply:
Great catch on persisting the number of confs across sessions! I implemented as you suggested. One thing that you'll probably notice is that I am not calling `(*Core).refreshUser` after increasing the number of confirmations because that field is a pointer, so changing it into the `dexConnection` is enough to get it updated. Is that ok?

Reply:
See #379, but yes, you will need to call `refreshUser` any time the `dexConnection.regConfirms` field is updated.

Reply:
Alright